Microsoft.Toolkit.Uwp.UI.Animations/Behaviors/ViewportBehavior.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Microsoft.Xaml.Interactivity;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations.Behaviors
```

Comment:
Can we instead move this to the `Microsoft.Toolkit.Uwp.UI.Behaviors` namespace and create a `Behaviors` folder here? https://github.com/windows-toolkit/WindowsCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.UI

Reply:
Done, move ```BehaviorBase``` and ```ViewportBehavior``` to ```Microsoft.Toolkit.Uwp.UI.Behaviors``` namespace.